docs/components/index.md

```diff
@@ -458,3 +458,132 @@ The following example shows using the `MarkupString` type to add a block of stat
     string myMarkup = "<p class='markup'>This is a <em>markup string</em>.</p>";
 }
 ```
+
+## Templated components
+
+*Templated components are available in Blazor 0.6.0 or later.*
```

Comment:
I don't think we need to version our content at this point. Things are moving too fast for that. I would remove this.